### PR TITLE
Translations for kiribati

### PR DESCRIPTION
**Pull request by nmadruga**

### CODE REVIEW

File: src/localization/tableStrings.js [hunk 1]
```diff
@@ -50,45 +50,51 @@ export const tableStrings = new LocalizedStrings({
     supply_quantity: 'SUPPLY THIS\nINVOICE',
     their_stock: 'THEIR\nSTOCK',
   },
-  gil: { // TODO add: batch, batch_expiry, our_stock, supply_quantity, supplier
```
Comment (andreievg) [inline src/localization/tableStrings.js]: I think `batch` is missing

File: src/localization/tableStrings.js [hunk 2]
```diff
@@ -50,45 +50,51 @@ export const tableStrings = new LocalizedStrings({
...
-    status: 'Teina\nngkai',
-    stock_on_hand: 'Mwaitin ae\niai ngkai',
-    suggested_quantity: 'Mwaiti ae\nkaantaningaki',
+    selected: 'RINEAKI',
+    snapshot_quantity: 'TE MWAITI',
+    status: 'TEINA\nNGKAI',
+    stock_on_hand: 'MWAITIN AE\nIAI NGKAI',
+    suggested_quantity: 'MWAITI AE\nKAANTANINGAKI',
+    supplier: 'KAMBANA NI KAOBWAI, PHARMACY',
+    supply_quantity: 'BEEBA N OOTA MAN TE PHARMACY',
```
Comment (andreievg) [inline src/localization/tableStrings.js]: Does `suppy_quantity` fit in the table ?

File: src/localization/tableStrings.js [hunk 3]
```diff
@@ -50,45 +50,51 @@ export const tableStrings = new LocalizedStrings({
...
-    name: 'Arana',
+    name: 'ARANA',
     no_change: 'Akea te bitaki',
     number_of_batches: 'Mwatin taian batch',
-    number_received: 'Mwaitin ae reke raoi',
-    number_sent: 'Mwaitin ae kanakoaki',
-    quantity: 'Mwaitina',
+    number_received: 'MWAITIN AE REKE RAOI',
+    number_sent: 'MWAITIN AE KANAKOAKI',
+    our_stock: 'MAITIN ARA BWAIN AORAKI INANON BAIRA',
```
Comment (andreievg) [inline src/localization/tableStrings.js]: Does our_stock fit in the table ?

File: src/localization/pageInfoStrings.js [hunk 4]
```diff
@@ -28,6 +27,7 @@ export const pageInfoStrings = new LocalizedStrings({
     entered_by: 'Entered By',
     entry_date: 'Entry Date',
     months_stock_required: 'Iraua te Namakaina?',
+    supplier: 'Kambana n oota, Pharmacy ke HC',
```
Comment (andreievg) [inline src/localization/pageInfoStrings.js]: Does `supplier` fit in the table ? and it seems quite long to describe 'supplier'

File: src/localization/tableStrings.js (hunk 1 above)
Comment (nmadruga) [reply src/localization/tableStrings.js]: Oh, that's probably a new one. I'll add as TODO. Thanks!

File: src/localization/tableStrings.js (hunk 3 above)
Comment (nmadruga) [reply src/localization/tableStrings.js]: Good questions. I think I missed those ones. Let me check it now...

File: src/localization/pageInfoStrings.js (hunk 4 above)
Comment (nmadruga) [reply src/localization/pageInfoStrings.js]: Yes, it is fitting: 
image

Comment (andreievg) [reply src/localization/pageInfoStrings.js]: Nice!

File: src/localization/tableStrings.js (hunk 3 above)
Comment (nmadruga) [reply src/localization/tableStrings.js]: I think it's good. What do you think?
Here it is, English and Gilbertese versions:
image

**Pull request merged**